client/package.json

```diff
@@ -5,6 +5,7 @@
     "normalize.css": "5.0.0",
     "preact": "^7.1.0",
     "preact-compat": "^3.9.2",
+    "react-markdown": "^2.4.4",
```

Comment:
ok, I will remove the import, and the dependency.
build wasn't working in master, and npm was complaining about react-markdown, so I tiried the easiest solution (npm install :  ).